prov/verbs/src/verbs_rma.c

```diff
@@ -400,6 +455,55 @@ fi_ibv_rdm_ep_rma_write(struct fid_ep *ep_fid, const void *buf, size_t len,
 	goto out;
 }
 
+static ssize_t
+fi_ibv_rdm_ep_rma_writemsg(struct fid_ep *ep, const struct fi_msg_rma *msg,
+		uint64_t flags)
+{
+	if(msg->iov_count == 1 && msg->rma_iov_count == 1) {
+		return fi_ibv_rdm_ep_rma_write(ep,
+					       msg->msg_iov[0].iov_base,
+					       msg->msg_iov[0].iov_len,
+					       msg->desc[0],
+					       msg->addr,
+					       msg->rma_iov[0].addr,
+					       msg->rma_iov[0].key,
+					       msg->context);
+	}
+
+	assert(0);
+	return -FI_EMSGSIZE;
+}
+
+static ssize_t
+fi_ibv_rdm_ep_rma_writev(struct fid_ep *ep, const struct iovec *iov, void **desc,
+		size_t count, fi_addr_t dest_addr, uint64_t addr, uint64_t key,
+		void *context)
+{
+	struct fi_rma_iov rma_iov = {
+		.addr = dest_addr,
+		.len = 0,
+		.key = key
+	};
+
+	size_t i;
+	for (i = 0; i < count; i++) {
+		rma_iov.len += iov[i].iov_len;
+	}
```

Comment:
As writev/readv are implemented via writemsg/readmsg, this is just packing args into rma_iov struct. This is not used now in writemsg/readmsg because they are implemented only for 1 iov so far. I believe it should make sense for iov count > 1.